openshift/templates/lup-admin/landuseplanning-admin-dev-deployment.yaml

```diff
@@ -0,0 +1,179 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: landuseplanning-admin-dev
+  namespace: e8b9ad-dev
+  labels:
+    app: landuseplanning-admin-dev
+    app.kubernetes.io/component: landuseplanning-admin-dev
+    app.kubernetes.io/instance: landuseplanning-admin-dev
+    app.kubernetes.io/part-of: landuseplanning-admin-dev
+  managedFields:
```

Comment:
`managedFields` tracks changes to a file over time. You can read more about it here: https://kubernetes.io/docs/reference/using-api/server-side-apply/

TLDR, you shouldn't need to copy this over too. Technically you'd be copying over the history of a different manifest (the DeploymentConfig) into this one.

Reply:
Removed